Mathlib/Data/Polynomial/Splits.lean

```diff
@@ -233,10 +233,10 @@ theorem splits_iff (f : K[X]) :
 #align polynomial.splits_iff Polynomial.splits_iff
 
 /-- This lemma is for polynomials over a field. -/
-theorem Splits.def {i : K →+* L} {f : K[X]} (h : Splits i f) :
+theorem Splits.def' {i : K →+* L} {f : K[X]} (h : Splits i f) :
```

Comment:
```suggestion
-- Adaptation note: 2024-03-15
-- Renamed to avoid the reserved name `Splits.def`.
theorem Splits.def' {i : K →+* L} {f : K[X]} (h : Splits i f) :
```